mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(number) {
+  let tax = calculateSalesTax(number);
```

Comment:
I really like the use of the tax variable to break down the stages of the function.